net/acme-common/Makefile

```diff
@@ -52,13 +50,13 @@ endef
 
 define Package/acme/postinst
 #!/bin/sh
-grep -q '/usr/bin/acme' /etc/crontabs/root 2>/dev/null && exit 0
-echo "0 0 * * * /usr/bin/acme get" >> /etc/crontabs/root
+grep -q '/etc/init.d/acme' /etc/crontabs/root 2>/dev/null && exit 0
+echo "0 0 * * * /etc/init.d/acme start" >> /etc/crontabs/root
 endef
 
 define Package/acme-common/prerm
 #!/bin/sh
-sed -i '\|/usr/bin/acme|d' /etc/crontabs/root
+sed -i '\|/etc/init.d/acme|d' /etc/crontabs/root
```

Comment:
Does this trigger on an update? Otherwise we need to add some logic to get rid of the old crontab entry...

Reply:
If it does, it should trigger the one in the old package?

Reply:
I've created a mock package to test it. Indeed the old prerm was run. So this should be good to merge.

BTW, I'm not sure if our uci-defaults necessary. I remember the reason for its existence is to support package being compiled into the firmware image, but if that's necessary, it seems to mean all packages should provide uci-defaults if they use postinst. But I don't see other packages do this, for example, mail/fdm.

Are you sure embedded packages won't run postinst? I don't compile firmware image myself, so I can't test right now.

Reply:
Alright, cool, thanks for checking. And yeah, I'm fairly sure postinst is not run by the image builder (it would have to execute in the context of the image, which may be an entirely different arch), so I think we'll need to keep it...